test/deform/linearized_deform_test.py

```diff
@@ -189,153 +191,132 @@ def exp_div_inv_disp(x, **kwargs):
 template = discr_space.element(hat_function)
 
 # Define the displacement field (x,y) -> eps * (x+y, 3xy)
-gradient_space = odl.ProductSpace(discr_space, 2)
-displacement_field = gradient_space.element(
-    [displacement_field0, displacement_field1])
+grad_space = odl.ProductSpace(discr_space, 2)
+disp_field = grad_space.element([disp_field0, disp_field1])
 
 
 # Test deformation for LinDeformFixedTempl
-def test_LinDeformFixedTempl():
-    deformed_template_analytic = discr_space.element(deformed_hat_function)
+def test_fixed_templ():
+    deform_templ_an = discr_space.element(deform_hat)
```

Comment:
the abreviation `an` for `analytic` is not obvious. Perhaps keep `analytic` written out